client/src/lib.rs

```diff
@@ -47,9 +47,6 @@ pub mod rpc_config {
 pub mod rpc_custom_error {
     pub use solana_rpc_client_api::custom_error::*;
 }
-pub mod rpc_deprecated_config {
-    pub use solana_rpc_client_api::deprecated_config::*;
-}
```

Comment:
This part of the diff is, I suppose, controversial, since there is no real way to warn users when they are using deprecated re-exports. Wdyt?

Reply:
seems like we made them type the word to use the symbols, so they know implicitly?

Reply:
True. That makes me feel better.